rules/windows/process_creation/proc_creation_win_etw_modification_cmdline.yml

```diff
@@ -23,7 +24,9 @@ logsource:
     product: windows
 detection:
     selection:
-        CommandLine|contains: 'COMPlus_ETWEnabled=0'
+        CommandLine|contains:
+            - 'COMPlus_ETWEnabled'
+            - 'COMPlus_ETWFlags'
```

Comment:
miss the '=0' part or I will have a trigger when set to 1

Reply:
Since it shouldn't be set via the CLI. I think it's better to know when it's even called. Don't you think?

Reply:
Title need to change in that case.

Reply:
Ok. Done.